bokehjs/src/coffee/models/tickers/mercator_ticker.coffee

```diff
@@ -0,0 +1,47 @@
+import {BasicTicker} from "./basic_ticker"
+import * as p from "core/properties"
+import {proj4, mercator} from "core/util/proj4"
+
+export class MercatorTicker extends BasicTicker
+  type: 'MercatorTicker'
+
+  @define {
+    dimension: [ p.LatLon ]
+  }
+
+  get_ticks_no_defaults: (data_low, data_high, cross_loc, desired_n_ticks) ->
+
+    if not @dimension?
+      throw Error("MercatorTickFormatter.dimension not configured")
+
+    if @dimension == "lon"
+      [proj_low,  _] = proj4(mercator).inverse([data_low,  cross_loc])
+      [proj_high, _] = proj4(mercator).inverse([data_high, cross_loc])
+    else
+      [_, proj_low ] = proj4(mercator).inverse([cross_loc, data_low ])
+      [_, proj_high] = proj4(mercator).inverse([cross_loc, data_high])
+
+    proj_cross_loc = _
```

Comment:
`_` is considered a throwaway value placeholder. This assignment should be moved to appropriate locations above.